app/lib/shared/versions.dart

```diff
@@ -10,6 +10,16 @@ import 'utils.dart' show isNewer;
 final String runtimeVersion = '2018.09.11';
 final Version semanticRuntimeVersion = new Version.parse(runtimeVersion);
 
+/// The version which marks the earliest version of the data which we'd like to
+/// keep during various GC processes. Data prior to this version is subject to
+/// delete (unless there is another rule in place to keep it).
+///
+/// Make sure that at least two versions are kept here as the next candidates
+/// when the version switch happens:
+/// - 2018.09.11
+/// - 2018.09.03
+final String gcBeforeRuntimeVersion = '2018.08.27';
```

Comment:
Depending on how much data each version consumes, and how much data we store, we could automate this. We could delete all data older than 6 months, for instance, if those versions are not in use. Alternatively, the above code could scan the directory and identity the N newest versions, and keep those and the older versions currently deployed.